### PR TITLE
Cleanup fix up for campaigns

### PR DESCRIPTION
**Pull request by n13**
Preparing to have an automated refund scheme after 1 cycle, so old invites get refunded

It will use this method to  clean up index ranges

### CODE REVIEW

**Review by IanMendozaJaimes — CHANGES_REQUESTED** [review]
In case of campaign invites, they should return their funds to the campaign before delete the invite

**Comment by n13** [comment]
> I think in case of campaign invites, they should return their funds to the campaign before delete the invite

Makes sense yes

**Comment by n13** [comment]
> I think in case of campaign invites, they should return their funds to the campaign before delete the invite

I am now sharing the code with cleanup, since it's so complicated - should work!

File: src/seeds.onboarding.cpp [hunk 1]
```diff
@@ -399,14 +405,11 @@ void onboarding::cleanup(uint64_t start_id, uint64_t max_id, uint64_t batch_size
...
-      iitr = invites.erase(iitr);
-      auto refitr = referrers.find(iitr->invite_id);
-      if (refitr != referrers.end()) {
-        referrers.erase(refitr);
-      }
-      count += 4;
+      name sponsor = iitr->sponsor;
+      checksum256 hash = iitr->invite_hash;
+      iitr++;
+      count += 8;
```
Comment (n13) [inline src/seeds.onboarding.cpp:411]: +8 because that's a heavyweight operation there ;)

**Review by IanMendozaJaimes — APPROVED** [review]
Looks great!

**Pull request merged**